Gemfile.lock

```diff
@@ -311,4 +330,4 @@ RUBY VERSION
    ruby 2.7.2p137
 
 BUNDLED WITH
-   2.2.0
+   2.2.6
```

Comment:
Will need to update the `BUNDLER_VERSION` in `Dockerfile`

Reply:
shall i change line 5 to ENV BUNDLER_VERSION=2.2.6 and re-push?

Reply:
pushed